core-ui/src/components/Extensibility/ResourceSchema.js

```diff
@@ -0,0 +1,97 @@
+import React, { useState, useCallback, useEffect } from 'react';
+import { isEmpty } from 'lodash';
+import { useTranslation } from 'react-i18next';
+
+import { createOrderedMap } from '@ui-schema/ui-schema/Utils/createMap';
+import { UIMetaProvider } from '@ui-schema/ui-schema/UIMeta';
+import {
+  UIStoreProvider,
+  createStore,
+  storeUpdater,
+} from '@ui-schema/ui-schema';
+import { injectPluginStack } from '@ui-schema/ui-schema/applyPluginStack';
+
+import formWidgets from './components-form';
+import { SchemaRulesInjector } from './SchemaRulesInjector';
+
+import { METADATA_SCHEMA } from './metadataSchema';
+
+const [firstPlugin, ...otherPlugins] = formWidgets.pluginStack;
+const widgets = {
+  ...formWidgets,
+  pluginStack: [firstPlugin, SchemaRulesInjector, ...otherPlugins],
+};
+
+function FormContainer({ children }) {
+  return <div container>{children}</div>;
```

Comment:
That container probably does nothing

Reply:
yeah, but ui-schema requires a wrapper component

Reply:
(the `container` flag is the key here)